GitSearch/Networking/HTTP/HTTPNetworkError.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  HTTPNetworkError.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/27/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import Foundation
+
+// The enumeration defines possible errrors to encounter during Network Request
+public enum HTTPNetworkError: String, Error {
+    
+    case parametersNil = "Error Found : Parameters are nil."
```

Comment:
"Error found" part seems to be excessive. Each error should explain only specific situation, general wording can be added on the level when you present error to the user. Also, to make it work properly, you should instead of adding String raw value to this enum, conform it to [`LocalizedError`](https://developer.apple.com/documentation/foundation/localizederror) and provide error description.

Reply:
I guess I can dive deeper in to the localization creating special files for that.
But for new I've just add localized description